Container Deployment Update (#1001)
spectrocloud/librarium: content/docs/04.5-devx/05-app-profile/02-container-deployment.md

# Container Deployment

> Palette App Mode supports the use of containers, a standard unit of software that packages code and all its dependencies to run applications quickly and reliably from one computing environment to another. They contain all the required executables, binary codes, libraries, and configuration files. As a result, containers are lightweight and portable with less overhead. To add a container tier to Palette Dev Engine App Profile follow the steps below. 

Reviewer: ```suggestion
Palette App Mode supports the use of containers, a standard unit of software that packages code and all its dependencies to run applications quickly and reliably from one computing environment to another. Containers contain all the required executables, binary codes, libraries, and configuration files. As a result, containers are lightweight and portable with less overhead. To add a container tier to Palette Dev Engine App Profile, follow the steps below. 
```
Clarified who they is.

> 

Reviewer: We need to clean up the intro. We need to explain why someone would benefit from this. Take a look at how we did the other How-to's.

![CleanShot 2022-11-28 at 07 18 35](https://user-images.githubusercontent.com/29551334/204300312-eb9c3330-8a9b-46be-a720-8ce781d51d40.png)

Author: done!

<br />

> # Prerequisite

Reviewer: We also need an app profile so we should list it here and link to the how to create an app profile page.

* [Spectro Cloud Palette account](https://www.spectrocloud.com/get-started/)

## How to Deploy Container to App Profile

> 1. Login to the Palette Dev Engine console.

Reviewer: ```suggestion
1. Login to [Palette](/devx#quickstartwithpaletteappmode).
```
This changes the language to be the same as the other How-to's related to app profile.

> 2. Select the **App Profiles** from the left **Main Menu** and click on the **New App Profile** button at the top right handside of the main screen. 

Reviewer: ```suggestion
2. Select **App Profiles** from the left **Main Menu** and click on the **New App Profile** button at the top right-hand side of the main screen. 
```

> 3. Provide the wizard with the following information and click on **Next** button after you have filled out the information.

Reviewer: ```suggestion
3. Provide the wizard with the following information and click on the **Next** button after you have filled out the information.
```

**Basic Information: **

|         Parameter           | Description  |
|-------------------------------|-----------------|
|Application Profile Name | A custom name for the App Profile|
|Description (optional)   | Description of the App Profile, if any | 
|Tag (optional)               | Tags on a cluster group are propagated to the cloud/datacenter environments.|

> 4. **Add Service**: From the available services select the **Container Deployment**, [Containers](https://www.docker.com/resources/what-container/) are methods of building, packaging, and deploying an application. A container includes the code, runtime, libraries, and all the dependencies required by a containerized workload. Container deployment deploys containers to their target environment. Review the [Palette Container Deployment](/devx/app-profile/container-deployment) to learn more about container deployment.

Reviewer: ```suggestion
4. **Add Service**: From the available services list, select **Container Deployment**. [Containers](https://www.docker.com/resources/what-container/) are methods of building, packaging, and deploying an application. A container includes the code, runtime, libraries, and all the dependencies required by a containerized workload. Container deployment deploys containers to their target environment. Review the [Palette Container Deployment](/devx/app-profile/container-deployment) to learn more about container deployment.
```

       argument: HOSTNAME
 
 

> 6. Click **Review** the information and complete the app profile creation wizard. Deploy more tiers to the profile per user requirements, and Palette Clusters can be deployed using the deployed app profile. 

Reviewer: ```suggestion
6. Click the **Review** button when you have filled out the information and are ready to conclude the wizard. You can add more services to the app profile if needed.
```

The last part of the sentence read very awkward.

> 

Reviewer: The next step should be to deploy the App Profile through the App page. In the App page, the user would deploy a new app by using the App Profile.

![CleanShot 2022-11-28 at 07 34 53](https://user-images.githubusercontent.com/29551334/204304145-d968acc9-9733-4d38-9b58-befd5a5175ab.png)

> ## Validation

Reviewer: Resma, the validation steps are not correct. The user should visit the Apps page from the Main Menu and check its status there.
